src/components/App.js

```diff
@@ -4,15 +4,16 @@ import '../css/App.css'
 import Nav from './Nav'
 import PosterGrid from './PosterGrid'
 import MovieDetails from './MovieDetails'
-import movieData from '../movieData'
+import endpoints from '../endpoints'
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
-      moviePosters: movieData.movies,
+      moviePosters: [],
```

Comment:
moviePosters, movieCards, movieDetailsless... I really do hate trying to name things.  Why does it insist on being so difficult?